lib/govuk_publishing_components.rb

```diff
@@ -20,6 +20,7 @@
 require "govuk_publishing_components/presenters/step_by_step_nav_helper"
 require "govuk_publishing_components/presenters/page_with_step_by_step_navigation"
 require "govuk_publishing_components/presenters/public_layout_helper"
+require "govuk_publishing_components/presenters/footer_helper"
```

Comment:
Can you rename the footer helper to match the component name? One of the things the component auditing does is list if a component has a helper, but it bases its search on what the component is named and currently it's not finding this file. Should work if it's `layout_footer_helper.rb`, I think.